include/openthread/channel_manager.h

```diff
@@ -72,6 +72,15 @@ extern "C" {
  */
 otError otChannelManagerRequestChannelChange(otInstance *aInstance, uint8_t aChannel);
 
+/**
+ * This method gets the channel for the last successfully requested channel change.
+ *
+ * @returns The channel for the last successfully requested channel change (last successful call to
```

Comment:
nit: just say "last successful call to..." rather than "last successfully requested..."